site/en/blog/help-css-nesting/index.md

```diff
@@ -0,0 +1,810 @@
+---
+layout: layouts/blog-post.njk
+title: Help CSS nesting pick syntax
+description: >
+  Two competing syntaxes need your help in determining which should be championed through to a spec candidate.
+subhead: >
+  Two competing syntaxes need your help in determining which should be championed through to a spec candidate.
+date: 2022-07-14
```

Comment:
update the date before merging :)

Reply:
i set it for 7-22 (tomorrow) but we can adjust as needed
thanks!